app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -207,6 +211,20 @@ class AwesomeBarView(
         searchSuggestionProviderMap = HashMap()
     }
 
+    private fun getSearchEngineSuggestionsHeader(components: Components): String? {
+        return components.core.store.state.search.selectedOrDefaultSearchEngine?.name?.let { name ->
```

Comment:
```suggestion
    private fun getSearchEngineSuggestionsHeader(name: String?): String? {
        return name?.let { name ->
```

this is a long call-chain from a function parameter. would it make more sense to just pass in the nullable name string?